test/cli_util_test.dart

```diff
@@ -36,4 +36,10 @@ void defineTests() {
       expect(isSdkDir(Directory(getSdkPath())), true);
     });
   });
+
+  group('applicationConfigHome', () {
+    test('returns a string', () {
+      expect(applicationConfigHome('dart'), isA<String>());
```

Comment:
Hmm - this is hard to test - but it seems we should be able to do better.

Could we at least confirm that the string ends with '${separator}dart'